src/Safe7579Launchpad.sol

```diff
@@ -286,13 +277,14 @@ contract Safe7579Launchpad is IAccount, SafeStorage, ISafeOp {
     )
         internal
         view
-        returns (bool validSig, uint48 validAfter, uint48 validUntil)
+        returns (bool, uint48, uint48)
     {
-        bytes memory operationData;
-        bytes calldata signatures;
-
-        (operationData, validAfter, validUntil, signatures) = _getSafeOp(userOp);
-
+        (
+            bytes memory operationData,
+            uint48 validAfter,
+            uint48 validUntil,
+            bytes calldata signatures
+        ) = _getSafeOp(userOp, SUPPORTED_ENTRYPOINT);
         bytes32 _hash = keccak256(operationData);
 
         InitData memory safeSetupCallData = abi.decode(userOp.callData[4:], (InitData));
```

Comment:
technically theres an issue in the next line with using `safeSetupCallData.threshold`: 
1. there is 1 owner and threshold of 1
2. a user supplies two sigs, one of which is from the owner
3. if the wrong sig comes first, then the validation will fail even if there is a correct sig that comes second

probably not an issue but wanted to mention this

Reply:
good should. should we require sorted instead?